stdlib/universe/sleep.go

```diff
@@ -32,21 +33,37 @@ var sleep = values.NewFunction(
 		Return:       semantic.Tvar(1),
 	}),
 	func(ctx context.Context, deps dependencies.Interface, args values.Object) (values.Value, error) {
-		v, ok := args.Get(vArg)
-		if !ok {
-			return nil, fmt.Errorf("missing argument %q", vArg)
-		}
-		d, ok := args.Get(durationArg)
-		if !ok {
-			return nil, fmt.Errorf("missing argument %q", durationArg)
-		}
-
-		if d.Type().Nature() == semantic.Duration {
-			dur := d.Duration()
-			time.Sleep(time.Duration(dur))
-		}
-		return v, nil
+		return interpreter.DoFunctionCallContext(Sleep, ctx, args)
 	},
 	// sleeping is a side effect
 	true,
 )
+
+func Sleep(ctx context.Context, args interpreter.Arguments) (values.Value, error) {
```

Comment:
Does this need to be public only for the test case? If so should we adjust the test case to use the `universe` package so it is private?

Reply:
It's not because of the test case. This is the same pattern I did with `runtime.Version`. I thought it was a good pattern because it made a public interface that could be used to access the native go functions.

If desired, I can hide it. I kind of like this pattern though. I can see how we might want to hide it though in circumstances where we might have a native go version but want to replace it with pure flux later.